modules/post/windows/manage/priv_migrate.rb

```diff
@@ -60,7 +61,9 @@ def run
   def get_pid(proc_name)
     processes = client.sys.process.get_processes
     processes.each do |proc|
-      return proc['pid'] if proc['name'] == proc_name
+      if proc['name'] == proc_name
```

Comment:
I think it would be easier to read if you combine the `if` conditions into one line here, i.e.:
```ruby
if proc['name'] == proc_name && proc['user'] != ""
```